### ISSUE TITLE
install Spyder-notebook for Spyder standalone application 

### ISSUE THREAD

**Issue by Jacques2101** [spyder-ide/spyder-notebook#346]
hi, how to install Spyder-notebook for Spyder standalone application ? 
thx

**Comment by jitseniesen** [comment]
@dalthviz Is it possible to install plugins like spyder-notebook if Spyder is installed with the stand-alone installer?

**Comment by ccordoba12** [comment]
My idea is to add spyder-terminal and/or spyder-notebook to the installers after they've been migrated to the new API and so better integrated with Spyder.

For now they have too many UX issues (especially spyder-notebook), so I think it's better to wait until Spyder 5.

**Comment by jitseniesen** [comment]
> For now they have too many UX issues (especially spyder-notebook), so I think it's better to wait until Spyder 5 is ready.

I agree, at least as far as spyder-notebook is concerned. Perhaps we need to wait even longer. The plugin looks nice, but when using it seriously too many rough edges become apparent. It still needs a bit more work.

@ccordoba12 Is the new API starting to stabilize sufficiently that I can start migrating the plugin to it?

**Comment by ccordoba12** [comment]
Not yet. I'd say mid February is a good time to start considering a migration.

**Comment by ObliviousMonkey** [comment]
I'm on Windows, with standalone Spyder 5.0.3 and was wondering the same thing. Has it progressed in the last months since?

**Comment by ccordoba12** [comment]
We first need to port Spyder-notebook to the Spyder 5 API (which is planned for this month). Then we could create installers on this repo with Spyder-notebook included, but that's a long term goal because right now we're quite busy with other things, sorry.

**Comment by gepcel** [comment]
Even with spyder-notebook added to the spyder installer, there's still the question of how to install another plugin to standalone spyder. I understand it's not supported yet. Is there a way to install plugins manually for standalone spyder in windows?

**Comment by bradwyatt** [comment]
Any update on this? Would be incredibly useful feature.

**Comment by ccordoba12** [comment]
@bradwyatt, perhaps we'll have a working solution during the second half of the year (but we can't make promises).
